client/state/notification-settings/test/actions.js

```diff
@@ -1,31 +1,24 @@
 /** @format */
 
-/**
- * External dependencies
- */
-import { expect } from 'chai';
-
 /**
  * Internal dependencies
  */
 import { requestNotificationSettings, updateNotificationSettings } from '../actions';
 import { NOTIFICATION_SETTINGS_REQUEST, NOTIFICATION_SETTINGS_UPDATE } from 'state/action-types';
 
-describe( 'actions', () => {
-	describe( '#requestNotificationSettings()', () => {
-		test( 'should return an action object', () => {
-			const action = requestNotificationSettings();
+describe( 'requestNotificationSettings()', () => {
+	test( 'should return an action object', () => {
+		const action = requestNotificationSettings();
 
-			expect( action ).to.eql( { type: NOTIFICATION_SETTINGS_REQUEST } );
-		} );
+		expect( action ).toEqual( { type: NOTIFICATION_SETTINGS_REQUEST } );
 	} );
+} );
 
-	describe( '#updateNotificationSettings()', () => {
-		test( 'should return an action object', () => {
-			const settings = {};
-			const action = updateNotificationSettings( { settings } );
+describe( 'updateNotificationSettings()', () => {
+	test( 'should return an action object', () => {
+		const settings = {};
+		const action = updateNotificationSettings( settings );
 
-			expect( action ).to.eql( { type: NOTIFICATION_SETTINGS_UPDATE, settings } );
-		} );
+		expect( action ).toEqual( { type: NOTIFICATION_SETTINGS_UPDATE, settings } );
```

Comment:
I'd say that these tests also became so trivial that they are useless. They just duplicate the action creator code.

Reply:
I agree with this, but this PR isn't introducing any new tests, it only migrates the ones that already exist. There are already a bunch of Calypso tests that only duplicate what they test and don't bring a lot of value, but I believe cleaning those up is material for another PR.